<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>439</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00ac14;&quot;&gt;Обов'язкові налаштування&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ff0000;&quot;&gt;Зміни налаштувань бази даних будуть активні після перезапуску програми!!!! &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>База даних                      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNameDB">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Таблиця працівників      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNameEmploeeT">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Таблиця з тваринами     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineNameAnimals">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Таблиця з статистикою </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineStatisticsT">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Драйвер Бази даних      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDriver">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Ім'я Сервера                   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineHostName">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="styleSheet">
      <string notr="true">color:#00AC14;</string>
     </property>
     <property name="text">
      <string>Стандартні вікна редагування таблиць</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Логін&lt;/span&gt; і &lt;span style=&quot; color:#aa0000;&quot;&gt;пароль&lt;/span&gt; потрібно використовувати тільки при підключені до локального сервера який вимагає дані параметри&lt;span style=&quot; color:#aa0000;&quot;&gt;!!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Логін    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineUserName">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color:#00AC14;</string>
         </property>
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linePasvord">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit{
border:1px solid #00AC14;

}
QLineEdit:hover{
border:2px solid #00AC14;
}
</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="ButtonStandartSetings">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background:#00AC14;
color:#FFE53E;
border:0px solid #FFFFFF;

}
QPushButton:hover{
background:#2DBE3B;
color:#FFE53E;
border:0px solid #FFFFFF;
}
QPushButton:pressed{
background:qlineargradient(spread:reflect, x1:0.433, y1:0, x2:0.438494, y2:0.500227, stop:0 rgba(255, 255, 255, 255), stop:0.365169 rgba(49, 196, 64, 255));
color:#FFE53E;
border:0px solid #FFFFFF;
}</string>
     </property>
     <property name="text">
      <string>Стандартні налаштування</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="ButtonSave">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:#00AC14;
color:#FFE53E;
border:0px solid #FFFFFF;

}
QPushButton:hover{
background:#2DBE3B;
color:#FFE53E;
border:0px solid #FFFFFF;
}
QPushButton:pressed{
background:qlineargradient(spread:reflect, x1:0.433, y1:0, x2:0.438494, y2:0.500227, stop:0 rgba(255, 255, 255, 255), stop:0.365169 rgba(49, 196, 64, 255));
color:#FFE53E;
border:0px solid #FFFFFF;
}</string>
       </property>
       <property name="text">
        <string>Зберегти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonClose">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:#00AC14;
color:#FFE53E;
border:0px solid #FFFFFF;

}
QPushButton:hover{
background:#2DBE3B;
color:#FFE53E;
border:0px solid #FFFFFF;
}
QPushButton:pressed{
background:qlineargradient(spread:reflect, x1:0.433, y1:0, x2:0.438494, y2:0.500227, stop:0 rgba(255, 255, 255, 255), stop:0.365169 rgba(49, 196, 64, 255));
color:#FFE53E;
border:0px solid #FFFFFF;
}</string>
       </property>
       <property name="text">
        <string>Вийти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
